contents/stacks_and_queues/code/python/stack_queue.py

```diff
@@ -0,0 +1,97 @@
+class Node:
+    def __init__(self, val):
+        self.value = val
+        self.next = None
+
+class Stack():
+    def __init__(self):
+        self.stack_list = None
+    
+    def push(self, val):
+        if self.stack_list == None:
+            self.stack_list=  Node(val)
```

Comment:
Slight formatting problem.
```suggestion
            self.stack_list = Node(val)
```

Reply:
ah yeah, forgot to use black XD, my mistake, ty

Reply:
done